pokedex/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "tacos",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^0.19.0",
+    "react": "^16.11.0",
+    "react-dom": "^16.11.0",
+    "react-redux": "^7.1.1",
+    "react-scripts": "3.2.0",
+    "redux": "^4.0.4",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.3.0"
```

Comment:
do you have styled components installed?  I tried deploying it but it's saying it's not there.  Do you have it globally?